index.html

```diff
@@ -1,25 +1,27 @@
-<<!DOCTYPE html>
+<!DOCTYPE html>
 <html>
- <head>
-    <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel="stylesheet">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./styles.css">
- </head>
-  <body>
-    <aside class="menu">
-      <h1>IdeaBox</h1>
-      <div class="border-box">
-        <h4 class="starred-ideas">Filter Starred Ideas</h4>
-        <button class="show-starred-ideas">Show Starred Ideas</button>
-      </div>
-    </aside>
-    <main>
-      <section class="top-section">
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <aside class="menu">
+    <h1>IdeaBox</h1>
+    <hr>
```

Comment:
Interesting use of the horizontal rule element! Did you do this to create a thematic break for accessibility devices?